### PR TITLE
Make tables of layers in QGIS Server option sortable. Fix #47044

### LINKED ISSUE #47044: Use the same display for WFS capabilities as for WMTS capabilities (Project Properties)
### Feature description

In the project properties window, in the QGIS Server tab it's possible to set the WFS capabilities (published, update, etc.). But the order of the layers in the list is random and when a project contains a large number of layers it can be complicated to find the desired layer.

Would it be possible to adopt the same display as for the WMTS capabilities settings? Or to order the layers alphabetically in the WFS capabilities panel?

image

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ismailsunni**
## Description

Make tables of layers in QGIS Server option sortable per #47044

Peek 2022-02-03 19-04 - WFS layer sortable

Funded by Camptocamp https://www.camptocamp.com

### CODE REVIEW

File: src/ui/qgsprojectpropertiesbase.ui [hunk 1]
```diff
@@ -1737,7 +1737,7 @@
                     <string/>
                    </property>
                    <property name="currentIndex">
-                    <number>0</number>
+                    <number>2</number>
```
Comment (DelazJ) [inline src/ui/qgsprojectpropertiesbase.ui:1740]: This one should stay at 0 to always open on the "services capabilities" tab and not WMTS. Right?
```suggestion
                    <number>0</number>
```

File: src/ui/qgsprojectpropertiesbase.ui [hunk 2]
```diff
@@ -269,7 +269,7 @@
           </sizepolicy>
          </property>
          <property name="currentIndex">
-          <number>0</number>
+          <number>10</number>
```
Comment (DelazJ) [inline src/ui/qgsprojectpropertiesbase.ui:272]:
```suggestion
          <number>0</number>
```
Start dialog on General tab and not QGIS Server

**Review by DelazJ — COMMENTED** [review]
leftover...

**Comment by ismailsunni** [comment]
Thanks, @DelazJ suggestion applied. :+1:

**Review by nyalldawson — APPROVED** [review]

**Pull request merged**